hazelcast/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -950,7 +934,7 @@ protected void handleMultiMapUnlock(String[] args) {
 
     // =======================================================
 
-    private void handStats(String[] args) {
+    private void handleStats(String[] args) {
```

Comment:
I didn't get why we left this as it is. Could you explain again?

Reply:
At first, I thought it would be good to print the unsupported operation message returned from ClientMultimapProxy. But, then I also agreed with you to remove this redundant handling to be consistent with other unsupported operations handling in the client console

Reply:
I see. I think similar to that, we should remove support for other operations as well, as they throw UnsupportedOperationExceptions

```
        } else if (iteratorStr.startsWith("q.")) {
            println(getQueue().getLocalQueueStats());
        } else if (iteratorStr.startsWith("l.")) {
            println(getList().getLocalListStats());
        } else if (iteratorStr.startsWith("s.")) {
            println(getSet().getLocalSetStats());
        }
```